src/app/plugins/marketplace.plugin.ts

```diff
@@ -98,6 +99,15 @@ export class MarketPlacePlugin extends Plugin {
         if (item?.length) {
           const user = msg.author;
           item += '\n' + user; //adds user to end of listing.
+
+          //Check if sold
+          const hasTargetReaction = msg.reactions.find(
+            (r) => r.emoji.name === this._TARGET_REACTION
+          );
+          if (hasTargetReaction) {
+            item += '\t SOLD';
+          }
+
```

Comment:
why include the listing at all if sold? and maybe you should check that the reaction is from the message.author

Reply:
People were complaining that they'd want to see sold listings. The author can still delete the message if they'd like

Reply:
I'll do the other part in a little bit

Reply:
Do you know why `console.log(hasTargetReaction.users)` prints an empty collection?

Reply:
probs some async thing